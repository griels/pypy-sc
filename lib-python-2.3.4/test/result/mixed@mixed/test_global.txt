command: /usr/bin/python /u/pedronis/PyPy/dist/pypy/tool/alarm.py 900 /u/pedronis/PyPy/dist/pypy/interpreter/py.py --oldstyle /u/pedronis/PyPy/dist/lib-python-2.3.4/test/test_global.py
run by: pedronis@ratthing-b246
sys.platform: linux2
sys.version_info: (2, 3, 5, 'final', 0)
startdate: Thu Apr 28 01:00:15 2005
pypy-revision: 11549
OUTPUT TEST
see output in: /u/pedronis/PyPy/dist/lib-python-2.3.4/test/result/test_global.out
============================================================
faking <type 'unicode'>
faking <type 'module'>
faking <type 'file'>
faking <type 'posix.stat_result'>
faking <type 'posix.statvfs_result'>
faking <type '_sre.SRE_Pattern'>
faking <type 'function'>
<test code>:2: SyntaxWarning: name 'a' is assigned to before global declaration
<test code>:2: SyntaxWarning: name 'b' is assigned to before global declaration
<test code>:2: SyntaxWarning: name 'x' is used prior to global declaration
<test code>:2: SyntaxWarning: name 'x' is assigned to before global declaration
FAILED: test output differs
========================== closed ==========================
execution time: 10.2659399509 seconds
exit status: 0
