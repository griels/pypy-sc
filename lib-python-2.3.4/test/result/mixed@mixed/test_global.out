test_global
should have raised SyntaxError: 
def wrong1():
    a = 1
    b = 2
    global a
    global b

should have raised SyntaxError: 
def wrong2():
    print x
    global x

should have raised SyntaxError: 
def wrong3():
    print x
    x = 2
    global x

as expected, no SyntaxError
